### DISCUSSION TITLE
Tailwind Prefix

### DISCUSSION THREAD

**Discussion by jason-nabooki** [inertiaui/modal#3] [General]
Hi Pascal,

Will the package work if we are using a tailwind prefix in the project that imports the package?

**Comment by pascalbaljet** [comment]
No, it won't. How is this usually solved with packages?

**Comment by jason-nabooki** [comment]
Hmm good question, I'm not too sure... 🤔 Packages compiling their own CSS is a common approach that would remove the dependency of Tailwind but there would be pros and cons to that.
